### DISCUSSION TITLE
tabbed with tabs logic

### DISCUSSION THREAD

**Discussion by jsupa** [beercss/beercss#355] [Q&A] [answered]
Hi, is it possible to use new .tabbed element with .tabs click logic ? So that when clicked it is switch to active ?

**Comment by KicKerBNU** [comment]
Hello @jsupa yes you can do it adding the active class

Here on this codepen if you inspect the element you will see the same behavior ( using beercss js )
https://codepen.io/kickerbnu/pen/bGBYMZR

Beercss already have a JS to control this but its not mandatory you can control by yourself if you want.

Have a good week 🍺

**Reply by RobinManoli** [reply]
It would be nice to have a working example, as tabs don't work out of the box even with the js. So an example with the data-ui tag, stating that it makes it work with beer.min.js

**Reply by leonardorafael** [reply, marked as answer]
@RobinManoli in https://github.com/beercss/beercss/releases/tag/v3.7.0 you can use `data-ui` or `ui` too.
